roles/ec2-ami-base/requirements.txt

```diff
@@ -0,0 +1,21 @@
+ansible==2.3.0.0
```

Comment:
this should be in the deploy repo, or in the base on the the roles repo so it can be common